main.go

```diff
@@ -123,17 +142,131 @@ func NewExporter(server string, timeout time.Duration) *Exporter {
 			nil,
 		),
 		evictions: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", "items_evicted_total"),
+			prometheus.BuildFQName(namespace, "", "evicted_total"),
 			"Number of valid items removed from cache to free memory for new items.",
 			nil,
 			nil,
 		),
 		reclaimed: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", "items_reclaimed_total"),
+			prometheus.BuildFQName(namespace, "", "reclaimed_total"),
 			"Number of times an entry was stored using memory from an expired entry.",
 			nil,
 			nil,
 		),
+		active_slabs: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "active_slabs_total"),
+			"Total number of slab classes allocated.",
+			nil,
+			nil,
+		),
+		malloced: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "malloced_bytes"),
+			"Number of bytes of memory allocated to slab pages.",
+			nil,
+			nil,
+		),
+		itemsNumber: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "items_number_total"),
+			"Total number of items currently stored in this slab class.",
```

Comment:
This seems to be a gauge and should not mention total in the help text or have the `_total` suffix. Counters must only be used for monotonically increasing values, the current number of items can go up and down.